src/c++/vdo/base/statistics.h

```diff
@@ -0,0 +1,289 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright 2023 Red Hat
+ *
+ * ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!!
+ *
+ * If you add new statistics, be sure to update the following files:
+ *
+ * ../../../c++/vdo/base/message-stats.c
+ * ../../../c++/vdo/base/pool-sysfs-stats.c
+ * ../../../c++/vdo/user/messageStatsReader.c
+ * ../../../c++/vdo/user/vdoStatsWriter.c
+ * ../../../perl/Permabit/Statistics/Definitions.pm
```

Comment:
These paths can be shortened. The vdo/base files are in this directory. The vd/user files  only need to be  ../user/[foo.c]

ALso pleaes remove the ATTENTION line  (lines 5-6)

Reply:
The version in the upstream submission shouldn't mention the non-kernel files. Maybe two comments, one inside an #ifdef ?

Reply:
Oh, Ken makes a good point. I'd maybe put the entire note about other files in an #ifdef. Especially since pool-sysfs-stats.c is also soon going to not be in the kernel build.